kitsune/community/jinja2/community/metrics.html

```diff
@@ -16,7 +16,7 @@
         <div class="inline-controls">
           <div>
             <select id="toggle-cohort-type">
-              <option value="contributor:kb:en-US">{{ _('en-US') }}</option>
+              <option value="contributor:kb:en-US">{{ '%(locale)s'|format(locale='en-US') }}</option>
```

Comment:
This can be just ``"en-US"``. As a string. I think the formatting is not needed and also a comment should be there about it not available for localization.